packages/nouns-webapp/src/components/NavWallet/NavWallet.module.css

```diff
@@ -165,13 +149,14 @@
   margin-top: 0.3rem;
 }
 
-.connectContentMobileText {
```

Comment:
is it okay that we are wiping out this class by renaming it? It's not being used anywhere else?

Reply:
nope! it's not used anywhere else. regarding the className, I just thought `connectContentMobileRoutes` will make more sense.